src/react-components/info-dialog.js

```diff
@@ -27,6 +27,28 @@ class InfoDialog extends Component {
 
     const loc = document.location;
     this.shareLink = `${loc.protocol}//${loc.host}${loc.pathname}`;
+    this.onKeyDown = this.onKeyDown.bind(this);
+    this.onContainerClicked = this.onContainerClicked.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener("keydown", this.onKeyDown);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("keydown", this.onKeyDown);
+  }
+
+  onKeyDown(e) {
+    if (e.key === "Escape") {
+      this.props.onCloseDialog();
+    }
+  }
+
+  onContainerClicked(e) {
+    if (e.currentTarget === e.target) {
```

Comment:
I noticed that the Help dialog from a Hub space (e.g., on [https://localhost:8080/hub.html?hub_id=0mnu4u5z3q2](https://localhost:8080/hub.html?hub_id=0mnu4u5z3q2)) does not get closed when clicking outside. can you change this to `if (e.target.closest(".dialog-overlay")) {`? (and likely need to move the `pointer-events: none` style.)